locales/pt-BR.json

```
@@ -96,14 +99,18 @@
    "pendingRecords" : "Registros Pendentes",
    "percent" : "{percent}%",
    "previousPage" : "Página Anterior",
+   "printExportRecords" : "Imprimir/Exportar Registros",
    "printRecords" : "Imprimir Registros",
    "progress" : "Progresso",
    "progressByType" : "Progresso por Tipo",
    "progressCurrentPeriod" : "Progresso no Período Atual",
    "questionName" : "Texto da Pergunta",
    "questionNamePlaceholder" : "Insira o texto da pergunta",
    "range" : "{rangeStart} – {RangeEnd}",
+   "rangeOnlyEnd" : "Grava até e incluindo (rangeEnd)",
```

they killed our bracket!
```suggestion
   "rangeOnlyEnd" : "Grava até e incluindo {rangeEnd}",
```